app.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-restricted-globals */
 /* eslint-disable max-classes-per-file */
 /* eslint-disable no-use-before-define */
```

Comment:
You have done a perfect work, however, we have a few points of improvement, talking about linter, it helps us clean and standardize our code regarding best practices and DRY method, to say so, I believe it is not a good practice to disable it in our code, I would strongly recommend you fix it up and remove the disabling lines as this will contribute to your professionalism and gain some good ways of writing a clean code in the near future, what do you think about it? 😃

Reply:
Fixed.